docfx.json

```diff
@@ -51,7 +51,8 @@
       "ms.topic": "managed-reference",
       "ms.service": "dotnet-api",
       "products": ["https://authoring-docs-microsoft.poolparty.biz/devrel/7696cda6-0510-47f6-8302-71bb5d2e28cf"],
-      "feedback_system": "Standard",
+      "feedback_system": "OpenSource",
+      "feedback_github_repo": "dotnet/dotnet-api-docs",
```

Comment:
There are a few more keys that are required for the feedback control to work correctly.

They are all listed here (note that the required ones are sprinkled throughout, not at the top): https://review.learn.microsoft.com/en-us/help/platform/user-feedback-enable-open-source?branch=main#configuration-metadata-for-open-source-feedback

The others you need are:

- `feedback_product_url`
- `open_source_feedback_contributorGuideUrl`
- `open_source_feedback_issueTitle`
- `open_source_feedback_issueUrl`
- `open_source_feedback_productName`